bash_completion

```diff
@@ -1989,7 +1989,7 @@ _cd()
 
     # Use standard dir completion if no CDPATH or parameter starts with /,
     # ./ or ../
-    if [[ -z ${CDPATH:-} || $cur == ?(.)?(.)/* ]]; then
+    if [[ ${CDPATH:-} || $cur == ?(.)?(.)/* ]]; then
```

Comment:
The test for `cd` is failing... Ah, this is a typo; `!` seems to have been missing here.

Reply:
Ah! Fixed, merging.